package.json

```diff
@@ -18,6 +18,7 @@
     "coverage": "codecov",
     "flow": "flow",
     "lint": "eslint ./src && prettier --check src/**/*.js --parser=flow && prettier --check src/**/*.ts --parser=typescript",
+    "lint-fix": "eslint ./src && prettier --write src/**/*.js --parser=flow && prettier --write src/**/*.ts --parser=typescript",
```

Comment:
Hey! I think we can drop this given that we have `lint-staged` and `husky` in place.

The reason why it didn't run for your previous file was because of a misconfiguration that was already fixed!
```suggestion
```